src/web/app/components/question-edit-form/question-edit-form.component.html

```diff
@@ -23,7 +23,7 @@
         </button>
         <button id="btn-save-question" type="button" class="btn btn-primary btn-sm" *ngIf="model.isEditable"
                 (click)="saveQuestionHandler(); $event.stopPropagation();" [disabled]="model.isSaving">
-          <tm-ajax-loading *ngIf="model.isSaving"></tm-ajax-loading><i class="fas fa-check"></i> Save
+          <tm-ajax-loading *ngIf="model.isSaving"></tm-ajax-loading><i class="fas fa-check"></i>Save
```

Comment:
You shouldn't remove the space here.
The space here is to separate the icon from the text, not for the loading icon.
You should just remove spaces that are directly beside the loading icons.

Reply:
Okay. Would it be possible to tell me the file names that require such change?

Reply:
I'll revert these changes first and then will remove the white space as per the file names that you'll provide.

Reply:
You can tell which spacing is for separating the loading icon vs which spacing is for separating the icon by looking at where the loading icon is placed.